kt/api-generator/src/main/kotlin/godot/codegen/traits/TypedTrait.kt

```diff
@@ -2,4 +2,5 @@ package godot.codegen.traits
 
 interface TypedTrait {
     val type: String?
+    val meta: String?
```

Comment:
Should that really be part of the TypeTrait when it's used for a single class and forced to null for everything else ?

Reply:
Yes, that's a type information, and should be used with TypedTrait extensions